Jenkinsfile

```diff
@@ -352,9 +352,9 @@ def packagingLinux(Map args = [:]) {
                 'linux/amd64',
                 'linux/386',
                 'linux/arm64',
+                'linux/armv7',
```

Comment:
Let's keep armv7 disabled by default. We don't publish those officially I think, so we can safe some CPU cycles on CI here.